### DISCUSSION TITLE
iOS Building Error

### DISCUSSION THREAD

**Discussion by klaydn** [codemagic-ci-cd/codemagic-docs#1702] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [x] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

When I attempt to build for iOS after having made updates to our app, as well as updating the flutter version and dependencies I am getting a pods error, originally it mentioned platoform issue, so I created a Podfile with the platform ios:, 11 and we are still having errors building, for some reason we not once got an error when we built for android. Apologies for any mistakes doing this post, its my first time here coming from Slack so still learning how discussions work. Any help is greatly appreciated!

### Expected results

App to Build

### Actual results

  Generating Pods project
      - Creating Pods project
      - Installing files into Pods project
        - Adding source files
        - Adding frameworks
        - Adding libraries
        - Adding resources
        - Linking headers
      - Installing Pod Targets
      - Installing Aggregate Targets
      - Generating deterministic UUIDs
      - Stabilizing target UUIDs
      - Running post install hooks
      - Writing Xcode project file to `Pods/Pods.xcodeproj`
      Cleaning up sandbox directory

    Integrating client projects
    [!] Could not automatically select an Xcode workspace. Specify one in your Podfile like so:

        workspace 'path/to/Workspace.xcworkspace'

    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer/user_project_integrator.rb:218:in `workspace_path'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer/user_project_integrator.rb:97:in `block in create_workspace'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer/user_project_integrator.rb:96:in `map'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer/user_project_integrator.rb:96:in `create_workspace'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer/user_project_integrator.rb:71:in `integrate!'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer.rb:929:in `block in integrate_user_project'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/user_interface.rb:64:in `section'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer.rb:925:in `integrate_user_project'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer.rb:185:in `integrate'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/installer.rb:170:in `install!'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/command/install.rb:52:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/lib/cocoapods/command.rb:52:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.12.1/bin/pod:55:in `<top (required)>'
    /Users/builder/.rbenv/versions/3.0.4/bin/pod:25:in `load'
    /Users/builder/.rbenv/versions/3.0.4/bin/pod:25:in `<main>'

Error output from CocoaPods:
↳

    [!] The Podfile does not contain any dependencies.

Error running pod install

Build failed :|
Failed to build for iOS

### Build id (optional)

646d39370483759fde733891

**Comment by icarusdust** [comment, marked as answer]
Hi @klaydn, I am reading the following error message:

`Unable to find a target named `RunnerTests` in project `Runner.xcodeproj`, did find `Runner`.`

Please check [this thread](https://www.reddit.com/r/iosdev/comments/13l5m9k/comment/jkqwlnv/?context=3) for a solution.

**Reply by klaydn** [reply]
Thank you, I did something similar to that thread, went made a new project and just copied my Runner.xcodeproj over with one or two minor changes. Also realized I had to do it on Android as well.
